src/test/java/com/mercadopago/client/payment/PaymentClientTest.java

```diff
@@ -633,12 +667,36 @@ private PaymentCreateRequest newCardPayment() {
             .ipAddress("127.0.0.1")
             .build();
 
+    PaymentFeeRequest fine =
+        PaymentFeeRequest.builder().type("fixed").value(new BigDecimal(2)).build();
+
+    PaymentFeeRequest interest =
+        PaymentFeeRequest.builder().type("fixed").value(new BigDecimal("0.03")).build();
```

Comment:
`fixed` não é suportado para interest
```suggestion
        PaymentFeeRequest.builder().type("percentage").value(new BigDecimal("0.03")).build();
```